src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -46,11 +45,6 @@ public InstallDialogViewModel(Dispatcher dispatcher, ILibraryCommandService libr
             List<IProvider> providers = new List<IProvider>();
             foreach (IProvider provider in deps.Providers.OrderBy(x => x.Id))
             {
-                if (provider.Id == UnpkgProvider.IdText)
```

Comment:
Can you explain this a bit to me. It looks like this change adds UnPkg back to the list of providers. Wasn't there some work where the default provider was supposed to be read from the libman.json file and used as the default?

Reply:
Making Unpkg the default provider is something that we want to do for preview 4. This change is for preview 3.